uftrace.c

```diff
@@ -338,6 +379,8 @@ static uint64_t parse_time(char *arg)
 		exp = 6; /* 10^6 */
 	else if (!strcasecmp(unit, "s") || !strcasecmp(unit, "sec"))
 		exp = 9; /* 10^9 */
+	else if (!strcasecmp(unit, "m") || !strcasecmp(unit, "min"))
+		return parse_min(val, decimal, nfore_zero);
```

Comment:
It'd be great if parse_min() can be implemented in a same way and in a same function.  IOW would it be really needed to handle it as a different function?